internal/manifest/os.go

```diff
@@ -537,6 +537,16 @@ func (p *OS) serialize() osbuild.Pipeline {
 	}
 
 	if p.OSTreeRef != "" {
+		pipeline.AddStage(osbuild.NewSystemdJournaldStage(
+			&osbuild.SystemdJournaldStageOptions{
+				Filename: "10-persistent.conf",
+				Config: osbuild.SystemdJournaldConfigDropin{
+					Journal: osbuild.SystemdJournaldConfigJournalSection{
+						Storage: osbuild.StoragePresistent,
+					},
+				},
+			},
+		))
```

Comment:
For correctness sake, it should be before the SELinux stage, but NewOSTreePrepTreeStage needs to be the last one.